### PR TITLE
Remove alpha feature test for EvenPodsSpread

### PR DESCRIPTION
**Pull request by notpad**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

EvenPodsSpread has been graduated to beta, it will be enabled by default, this test can be removed now.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by draveness — COMMENTED** [review]
/lgtm

/assign @huang-wei

File: pkg/scheduler/apis/config/testing/compatibility_test.go [hunk 1]
```diff
@@ -1293,40 +1293,6 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 				Ignorable:        true,
 			}},
 		},
-		{
```
Comment (Huang-Wei) [inline pkg/scheduler/apis/config/testing/compatibility_test.go]: LGTM. One nit: can you modify the test "enable alpha feature ResourceLimitsPriorityFunction" to be "enable alpha feature ResourceLimitsPriorityFunction and disable beta feature EvenPodsSpread":

```diff
diff --git a/pkg/scheduler/apis/config/testing/compatibility_test.go b/pkg/scheduler/apis/config/testing/compatibility_test.go
index b3662be76c8..86337b47b46 100644
--- a/pkg/scheduler/apis/config/testing/compatibility_test.go
+++ b/pkg/scheduler/apis/config/testing/compatibility_test.go
@@ -1294,41 +1294,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			}},
 		},
 		{
-			name: "enable alpha feature EvenPodsSpread",
-			JSON: `{
-		  "kind": "Policy",
-		  "apiVersion": "v1",
-		  "predicates": [
-			{"name": "EvenPodsSpread"}
-		  ],
-		  "priorities": [
-			{"name": "EvenPodsSpreadPriority",   "weight": 2}
-		  ]
-		}`,
-			featureGates: map[featuregate.Feature]bool{
-				features.EvenPodsSpread: true,
-			},
-			wantPlugins: map[string][]config.Plugin{
-				"QueueSortPlugin": {{Name: "PrioritySort"}},
-				"PreFilterPlugin": {
-					{Name: "PodTopologySpread"},
-				},
-				"FilterPlugin": {
-					{Name: "NodeUnschedulable"},
-					{Name: "TaintToleration"},
-					{Name: "PodTopologySpread"},
-				},
-				"PreScorePlugin": {
-					{Name: "PodTopologySpread"},
-				},
-				"ScorePlugin": {
-					{Name: "PodTopologySpread", Weight: 2},
-				},
-				"BindPlugin": {{Name: "DefaultBinder"}},
-			},
-		},
-		{
-			name: "enable alpha feature ResourceLimitsPriorityFunction",
+			name: "enable alpha feature ResourceLimitsPriorityFunction and disable beta feature EvenPodsSpread",
 			JSON: `{
 		  "kind": "Policy",
 		  "apiVersion": "v1",
@@ -1338,6 +1304,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 		  ]
 		}`,
 			featureGates: map[featuregate.Feature]bool{
+				features.EvenPodsSpread:                 false,
 				features.ResourceLimitsPriorityFunction: true,
 			},
 			wantPlugins: map[string][]config.Plugin{

```

**Review by draveness — COMMENTED** [review]
/lgtm

File: pkg/scheduler/apis/config/testing/compatibility_test.go (hunk 1 above)
Comment (notpad) [reply pkg/scheduler/apis/config/testing/compatibility_test.go]: done

**Pull request merged**